src/resultsdisassemblypage.cpp

```diff
@@ -573,11 +573,17 @@ void ResultsDisassemblyPage::showDisassembly()
         if (!objdump.isEmpty())
             return objdump;
 
+#ifndef __aarch64__
         if (m_arch.startsWith(QLatin1String("armv8")) || m_arch.startsWith(QLatin1String("aarch64"))) {
             return QStringLiteral("aarch64-linux-gnu-objdump");
         }
-        const auto isArm = m_arch.startsWith(QLatin1String("arm"));
-        return isArm ? QStringLiteral("arm-linux-gnueabi-objdump") : QStringLiteral("objdump");
+#endif
+#ifndef __arm__
+        if (m_arch.startsWith(QLatin1String("arm"))) {
```

Comment:
I assume `__arm__` will be defined together with `__aarch64__`, right?

generally though, I wonder if we should instead just do something like this instead:

```
      if (m_arch.startsWith(QLatin1String("armv8")) || m_arch.startsWith(QLatin1String("aarch64"))) {
            if (auto aarch64Objdump = QStandardPaths::findExecutable(QStringLiteral("aarch64-linux-gnu-objdump")); !aarch64Objdump.isEmpty()))
                return aarch64Objdump;
        }
```

and dito for the other one - this should always work and is independent of the compile time macros? what do you think?